src/components/IndiaStateMap/IndiaStateMap.js

```diff
@@ -91,7 +94,10 @@ const IndiaStateMap = ({ isMobile, theme }) => {
         activeAttribute={activeAttribute}
         currentIndex={currentIndex}
         handleSliderUpdate={handleSliderUpdate}
-        max={stateDataManager.getLatest().index}></IndiaMapControls>
+        max={stateDataManager.getLatest().index}
```

Comment:
I think we are not using `max` in the component and it can be safely removed

Reply:
Done

Reply:
Max is required for date slider